doc/tutorial.md

```diff
@@ -238,7 +238,21 @@ Basically, this option makes training iteration faster than `--batch-count seq`.
 
     This creates the minibatch that has the maximum number of input, output and input+output frames under 800, 100 and 900, respectively. You can set one of `--batch-frames-xxx` partially. Like `--batch-bins`, this option makes training iteration faster than `--batch-count seq`. If you already has the best `--batch-seqs x` config, try `--batch-frames-in $((x * (mean(ilen) * idim)) --batch-frames-out $((x * mean(olen) * odim))`.
 
+### How to use finetuning
 
+ESPnet currently supports two finetuning operations: transfer learning (1.x) and freezing (2.).
+
+1.1. Transfer learning option is split between encoder initialization (`--enc-init`) and decoder initialization (`--dec-init`). However, the same model can be specified for both options. Each one taking a snapshot or model path.
```

Comment:
Can you clarify this sentence a bit more (with some examples?)? 
"Each one taking a snapshot or model path."

Reply:
Is that okay like this?

Reply:
OK for me!